lib/manifest.c

```diff
@@ -107,19 +110,31 @@ rpmRC rpmReadPackageManifest(FD_t fd, int * argcPtr, char *** argvPtr)
 
 	/* Concatenate next line in buffer. */
 	*se = '\0';
-	argvAdd(&sb, s);
-    }
 
-    s = argvJoin(sb, " ");
+	poptParseArgvString(s, &pac, &pav);
+	for (j = 0; j < pac; j++)
+	    argvAdd(&sb, pav[j]);
```

Comment:
I wonder if this strange little dance is worth it to preserve multiple arguments on a single manifest line. It is documented behavior so from that POV this is the right thing to do, just wondering if the behavior itself is sane :sweat_smile: I certainly don't remember ever seeing a manifest with more than a single argument per line - and most people have never so much as heard of rpm manifest files...

Reply:
Truly, I've left it there just for the sake of compatibility, and was actually going to remove most of this anyway, by making `rpmGlob()` extend the passed arglist instead of replacing it on each call, in another PR.